### DISCUSSION TITLE
ROG Community Admin about Ghelper

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#2070] [General]
Hi,

While some of the Asus Customer Agents sometimes advise using Ghelper, [it seems that the ROG Community admin has a different opinion.](https://rog-forum.asus.com/t5/tuf-asus-gaming-notebooks/what-are-these-new-effects-i-use-my-computer-for-work/td-p/996145)

It's his role to warn of risks and that's normal. But it may be a little exaggerated regarding Ghelper. Other overclocking software can also compromise hardware security, much more so than Ghelper !

He probably also wants to preserve ASUS interests so that people continue to use Armory Crate. For this, ASUS should work more seriously on AC rather than offering useless things and bugs with each update.

In any case, this will not change my opinion on Ghelper which remains and will remain a fantastic software. \o/ 
And I would continue to recommend it. 

PS: Have you had any discussions with ASUS Seerge ? :)

**Comment by seerge** [comment]
@Kikimahe nope, I have never been contacted by ASUS. But if they have any tips or suggestions, I'm always open for that :)

As for that remark - G-Helper allows you to set same predefined modes as Armoury, using Armoury's own endpoints. So it's literally **same** modes. Optional custom fan curves or power limits are also go via Armoury's endpoints and are still and always handled / controlled by BIOS. 

After all there are lot of people who use Linux on Asus laptops. Where you don't have any vendor software by definition. And Linux again uses [same endpoints to set modes](https://github.com/torvalds/linux/blob/4f5e5092fdbf5cec6bedc19fbe69cce4f5f08372/include/linux/platform_data/x86/asus-wmi.h) :) 

P.S. And app is **open source and is fully transparent** - so remarks about security just sound strange.

**Comment by Kikimahe** [comment]
I think they see that a lot of people are advising Ghelper and it's starting to worry them.

Instead of pointing the finger at Ghelper, Asus should question why people are looking for an alternative.
Quite simply because Armory Crate is not a good stable software solution.

 It's up to them to work to make AC better if they want to keep users. 
And there is a lot of work... :)

**Comment by IceStormNG** [comment]
It's just typical marketing bs. Security is a great and somehow abstract term to tell people, that don't know any better, to not use something. It's not just ASUS pulling such a thing.

Of course ASUS wants you to use AC. It is not only their software, but it also contains ads and stuff from some partners and has a lot of telemetry that they likely do not want to miss out.

And yes. ASUS should just have a look and check why people do not want AC. But they seemingly do not care. Software has never been ASUS' strength, and instead of trying to improve they just make it worse. AC on the Ally is surprisingly not that bad, but AC on laptops and especially on desktops is a nightmare. Even worse: When you need AC for just a peripheral device. Having a giant 2+GB App with 9 background services that are never idle and constantly spam messages around is sure great. Especially for laptops when on battery.

AC is also large (in terms of screen space usage), bloated, and laggy. Honstly, it is a tool to set configurations. Slow animations and fancy design are really out of place here if you ask me.

It is sure interesting how people trust closed source apps from companies more than open source apps in terms of security and privacy.
